### PR TITLE
feat(config): add Alloy (Zipato) devices

### PR DESCRIPTION
**Pull request by apella12**
In response to #6323 and #6324. These devices are neo (0x0258) clones, so I copied those configs and made adjustments based on the manual provided by the poster.

PR description here
[alloycontact.PDF](https://github.com/zwave-js/node-zwave-js/files/12738320/alloycontact.PDF)
[alloy5in1sensor.PDF](https://github.com/zwave-js/node-zwave-js/files/12738321/alloy5in1sensor.PDF)

### CODE REVIEW

**Comment by AlCalzone** [comment]
Where did you find the device labels? On their website, I only find the [Contact Sensor 1 ](https://smartrent.com/hardware/sensors/alloy-smarthome-contact-sensor/)(?), which seems to be based on the NAS-DS01Z. They call it `AH-NAS-DS01Z`, so I guess the new one would be called `AH-NAS-DS07Z`.

Similarly, the 5-in-1 motion sensor would likely be called `AH-NAS-PD07ZU1`.

File: packages/config/config/devices/0x0131/ne-nas-pd07u1.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
+{
+	"manufacturer": "Zipato",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0131/ne-nas-pd07u1.json]: While this is called `manufacturer`, we're actually using it for the brand name.
```suggestion
	"manufacturer": "Alloy",
```

File: packages/config/config/devices/0x0131/ne-nas-pd07u1.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+	"manufacturerId": "0x0131",
+	"label": "NE-NAS-PD07U1",
+	"description": "Alloy 5 in 1 PIR Motion Sensor",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0131/ne-nas-pd07u1.json]:
```suggestion
	"description": "5 in 1 PIR Motion Sensor",
```

File: packages/config/config/devices/0x0131/ne-nas-pd07u1.json [hunk 3]
```diff
@@ -0,0 +1,161 @@
+{
+	"manufacturer": "Zipato",
+	"manufacturerId": "0x0131",
+	"label": "NE-NAS-PD07U1",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0131/ne-nas-pd07u1.json]:
```suggestion
	"label": "AH-NAS-PD07U1",
```
probably?

Comment (AlCalzone) [reply packages/config/config/devices/0x0131/ne-nas-pd07u1.json]: Also rename the file if we do this.

File: packages/config/config/devices/0x0131/ne-neo-ds07z.json [hunk 4]
```diff
@@ -0,0 +1,124 @@
+{
+	"manufacturer": "Zipato",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0131/ne-neo-ds07z.json]:
```suggestion
	"manufacturer": "Alloy",
```

File: packages/config/config/devices/0x0131/ne-neo-ds07z.json [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+	"manufacturerId": "0x0131",
+	"label": "NE-NEO-DS07Z",
+	"description": "Alloy Contact Sensor 2",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0131/ne-neo-ds07z.json]:
```suggestion
	"description": "Contact Sensor 2",
```

File: packages/config/config/devices/0x0131/ne-neo-ds07z.json [hunk 6]
```diff
@@ -0,0 +1,124 @@
+{
+	"manufacturer": "Zipato",
+	"manufacturerId": "0x0131",
+	"label": "NE-NEO-DS07Z",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0131/ne-neo-ds07z.json]:
```suggestion
	"label": "AH-NEO-DS07Z",
```
probably? If so, also rename the file.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0131/ne-nas-pd07u1.json (hunk 1 above)
Comment (apella12) [reply packages/config/config/devices/0x0131/ne-nas-pd07u1.json]: Ok, noted.  Wasn't sure if you wanted to align with the 0x0131 description

File: packages/config/config/devices/0x0131/ne-nas-pd07u1.json (hunk 2 above)
Comment (apella12) [reply packages/config/config/devices/0x0131/ne-nas-pd07u1.json]: Ok

File: packages/config/config/devices/0x0131/ne-neo-ds07z.json (hunk 4 above)
Comment (apella12) [reply packages/config/config/devices/0x0131/ne-neo-ds07z.json]: ditto

File: packages/config/config/devices/0x0131/ne-neo-ds07z.json (hunk 5 above)
Comment (apella12) [reply packages/config/config/devices/0x0131/ne-neo-ds07z.json]: ok

File: packages/config/config/devices/0x0131/ne-neo-ds07z.json (hunk 6 above)
Comment (apella12) [reply packages/config/config/devices/0x0131/ne-neo-ds07z.json]: ditto

File: packages/config/config/devices/0x0131/ne-nas-pd07u1.json (hunk 3 above)
Comment (apella12) [reply packages/config/config/devices/0x0131/ne-nas-pd07u1.json]: ok, How is NE or AH determined? I just copied from existing examples

Comment (AlCalzone) [reply packages/config/config/devices/0x0131/ne-nas-pd07u1.json]: I don't know where the other label came from tbh. I just found the Alloy website where some of their older devices are listed with the device label that just looks like a prefixed version of the Shenzhen Neo variants.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**